src/extensions/replay/sessionrecording-utils.ts

```diff
@@ -61,6 +61,7 @@ export declare type recordOptions<T> = {
     plugins?: RecordPlugin[]
     mousemoveWait?: number
     keepIframeSrcFn?: KeepIframeSrcFn
+    root?: HTMLElement
```

Comment:
should we document here why we're adding this config?
do all flutter users have to set it?